code/modules/reagents/reagent_containers/patch.dm

```diff
@@ -44,3 +44,9 @@
 	desc = "Helps with brute and burn injuries."
 	list_reagents = list(/datum/reagent/medicine/synthflesh = 20)
 	icon_state = "bandaid_both"
+
+/obj/item/reagent_containers/pill/patch/nicotine
```

Comment:
fucks me up that patches are still pathed under /pill, thought for a minute that you would just pop the nicotine patches in your mouth 😆

Reply:
ya i thought it was wack too, they can actually fit in pill bottles bc of it. i saw a piece of code that I think prevented people from being able to eat them so thats whatever